From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Zinc Authors <zinc@users.noreply.github.com>
Date: Tue, 27 Oct 2026 11:45:25 +0000
Subject: [PATCH 1/6] Zinc: Branding

Name the app Zinc and use the Zinc launcher icon (overlay drawables
zinc_launcher_{background,foreground,monochrome}) instead of the Chromium
logo. The package name is set through the chrome_public_manifest_package
GN arg in args/zinc_*.gn.

diff --git a/chrome/android/BUILD.gn b/chrome/android/BUILD.gn
index 271801af..78140a27 100644
--- a/chrome/android/BUILD.gn
+++ b/chrome/android/BUILD.gn
@@ -136,6 +136,9 @@ if (_is_default_toolchain) {
       "java/res_base/anim/no_anim.xml",
       "java/res_base/drawable/ic_launcher.xml",
       "java/res_base/drawable/ic_launcher_round.xml",
+      "java/res_base/drawable/zinc_launcher_background.xml",
+      "java/res_base/drawable/zinc_launcher_foreground.xml",
+      "java/res_base/drawable/zinc_launcher_monochrome.xml",
       "java/res_base/font/chrome_google_sans.xml",
       "java/res_base/font/chrome_google_sans_bold.xml",
       "java/res_base/font/chrome_google_sans_medium.xml",
diff --git a/chrome/android/java/res_base/drawable/ic_launcher.xml b/chrome/android/java/res_base/drawable/ic_launcher.xml
index e5a868e4..2265aa90 100644
--- a/chrome/android/java/res_base/drawable/ic_launcher.xml
+++ b/chrome/android/java/res_base/drawable/ic_launcher.xml
@@ -5,8 +5,9 @@ Use of this source code is governed by a BSD-style license that can be
 found in the LICENSE file.
 -->
 
+<!-- Zinc: brand icon instead of the Chromium logo. -->
 <adaptive-icon xmlns:android="http://schemas.android.com/apk/res/android" >
-    <background android:drawable="@android:color/white"/>
-    <foreground android:drawable="@mipmap/layered_app_icon"/>
-    <monochrome android:drawable="@drawable/themed_app_icon"/>
+    <background android:drawable="@drawable/zinc_launcher_background"/>
+    <foreground android:drawable="@drawable/zinc_launcher_foreground"/>
+    <monochrome android:drawable="@drawable/zinc_launcher_monochrome"/>
 </adaptive-icon>
diff --git a/chrome/android/java/res_base/drawable/ic_launcher_round.xml b/chrome/android/java/res_base/drawable/ic_launcher_round.xml
index 2159da0b..2265aa90 100644
--- a/chrome/android/java/res_base/drawable/ic_launcher_round.xml
+++ b/chrome/android/java/res_base/drawable/ic_launcher_round.xml
@@ -5,8 +5,9 @@ Use of this source code is governed by a BSD-style license that can be
 found in the LICENSE file.
 -->
 
+<!-- Zinc: brand icon instead of the Chromium logo. -->
 <adaptive-icon xmlns:android="http://schemas.android.com/apk/res/android" >
-    <background android:drawable="@mipmap/layered_app_icon_background"/>
-    <foreground android:drawable="@mipmap/layered_app_icon_foreground"/>
-    <monochrome android:drawable="@drawable/themed_app_icon"/>
+    <background android:drawable="@drawable/zinc_launcher_background"/>
+    <foreground android:drawable="@drawable/zinc_launcher_foreground"/>
+    <monochrome android:drawable="@drawable/zinc_launcher_monochrome"/>
 </adaptive-icon>
diff --git a/chrome/android/java/res_chromium_base/values/channel_constants.xml b/chrome/android/java/res_chromium_base/values/channel_constants.xml
index cc1f2a30..884235d1 100644
--- a/chrome/android/java/res_chromium_base/values/channel_constants.xml
+++ b/chrome/android/java/res_chromium_base/values/channel_constants.xml
@@ -7,8 +7,8 @@ found in the LICENSE file.
 
 <resources xmlns:android="http://schemas.android.com/apk/res/android">
     <!-- The application name displayed to the user. -->
-    <string name="app_name" translatable="false">Chromium</string>
-    <string name="bookmark_widget_title" translatable="false">Chromium bookmarks</string>
-    <string name="search_widget_title" translatable="false">Chromium search</string>
-    <string name="quick_action_search_widget_title" translatable="false">Chromium quick action search</string>
+    <string name="app_name" translatable="false">Zinc</string>
+    <string name="bookmark_widget_title" translatable="false">Zinc bookmarks</string>
+    <string name="search_widget_title" translatable="false">Zinc search</string>
+    <string name="quick_action_search_widget_title" translatable="false">Zinc quick action search</string>
 </resources>
